lib_eio_windows/test/test_fs.ml

```diff
@@ -119,80 +114,61 @@ let test_append env () =
   Path.save ~create:`Never ~append:true test_file "2nd-write";
   Alcotest.(check string) "append" "1st-write-original2nd-write" (Path.load test_file)
 
-let test_mkdir env () =

```

Comment:
Why did you move `test_mkdir` from here to further down the file? It is an extra diff that doesn't seem to be necessary, but perhaps I am mistaken?

Reply:
No, i think it must have been an oversight from me.